Jamulus.pro

```diff
@@ -238,18 +238,53 @@ win32 {
     LIBS += -framework AVFoundation \
         -framework AudioToolbox
 } else:android {
-    ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+    # By default build for all the ABIs
+    # ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+
+    # get ANDROID_ABIS from environment - passed directly to qmake
+    ANDROID_ABIS = $$getenv(ANDROID_ABIS)
```

Comment:
From where will these come when running `make distclean; qmake ...`?

There should be a set of defaults that can be overridden from the build chain.